docker/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM node:10.8.0
+
+RUN wget https://www.python.org/ftp/python/3.6.3/Python-3.6.3.tar.xz && \
+    tar xJf Python-3.6.3.tar.xz && \
+    cd Python-3.6.3 && \
+    ./configure && \
+    make && \
+    make install
+
+COPY . /home/node/kamu
+WORKDIR /home/node/kamu
+
+RUN python3 -m venv kamu &&  \
```

Comment:
@Djuki when we are in a container that runs isolate processes, it might be not necessary to create a virtualenv.